testing/test_pytest_html.py

```diff
@@ -1005,12 +1005,32 @@ def test_css(self, testdir, recwarn, colors):
             assert str(v["path"]) in html
             assert v["style"] in html
 
-    def test_css_invalid(self, testdir, recwarn):
+    @pytest.mark.parametrize(
+        "files",
+        [
+            "style.css",
+            ["abc.css", "xyz.css"],
+            "testdir.makefile('.css', * {color: 'white'}",
+        ],
+    )
+    def test_css_invalid(self, testdir, recwarn, files):
+        assert files is not None
```

Comment:
Is this debug code? `files` should never be `None` based on your parametrization